src/display/node_stream.js

```diff
@@ -362,8 +362,13 @@ class PDFNodeStreamRangeReader extends BaseRangeReader {
 class PDFNodeStreamFsFullReader extends BaseFullReader {
   constructor(stream) {
     super(stream);
+
     let path = decodeURI(this._url.path);
 
+    if (/^file:\/\/\/[a-zA-Z]:\//.test(this._url.href)) {
```

Comment:
This regex looks like the same one below. Can we cache it at the top of the as a constant, e.g., something like:

```javascript
const fileUriRegex = /^file:\/\/\/[a-zA-Z]:\//
```

and then use `fileUriRegex` here?